tasks.py

```diff
@@ -512,19 +519,33 @@ def type_schema(
         schema_dir = GX_ROOT_DIR / "experimental" / "datasources" / "schemas"
         for name in _SourceFactories.type_lookup.type_names():
             model = _SourceFactories.type_lookup[name]
+
+            if (
+                model in {*PandasDatasource.asset_types, PandasDatasource}
+                and _PANDAS_SCHEMA_VERSION != pandas.__version__
+            ):
+                print(
+                    f"🙈  {name} - was generated with pandas {_PANDAS_SCHEMA_VERSION}; skipping"
+                )
+                continue
```

Comment:
@NathanFarmer @billdirks this logic will stop `invoke schemas --sync` from generating "bad" pandas schemas.